tests/system/Database/Live/ForgeTest.php

```diff
@@ -34,8 +34,16 @@ final class ForgeTest extends CIUnitTestCase
 
     protected function setUp(): void
     {
-        parent::setUp();
         $this->forge = Database::forge($this->DBGroup);
```

Comment:
this helps when testing locally if something goes wrong and the test doesn't finish.